gbpservice/neutron/nsf/core/main.py

```diff
@@ -0,0 +1,705 @@
+import os
+import time
+import sys
+import copy
+import threading
+
+import eventlet
+eventlet.monkey_patch()
+
+from multiprocessing.queues import Queue
+from Queue import Empty, Full
+from multiprocessing import Process, Queue, Lock
+import multiprocessing as multiprocessing
+
+from neutron.agent.common import config
+from neutron.common import config as common_config
+from neutron import context
+from neutron.agent import rpc as agent_rpc
+
+from gbpservice.neutron.nsf.core import cfg as core_cfg
+from gbpservice.neutron.nsf.core import lb as core_lb
+from gbpservice.neutron.nsf.core import threadpool as core_tp
+from gbpservice.neutron.nsf.core import periodic_task as core_periodic_task
+from gbpservice.neutron.nsf.core import queue as core_queue
+
+from neutron.common import rpc as n_rpc
+
+from oslo_log import log as logging
+from oslo_config import cfg
+from oslo_service import periodic_task as oslo_periodic_task
+from oslo_service import loopingcall
+from oslo_service import service as os_service
+
```

Comment:
It would be confusing to call some as oslo_, some as os_ and some others without any prefix etc. Can we call it oslo_service instead of os_service?